app/jobs/redis_set_job_spawner.rb

```diff
@@ -0,0 +1,34 @@
+# An ActiveJob designed to spawn a bunch of subclasses of RedisSetJob, using
+# the contents of a Redis set to generate the jobs.
+class RedisSetJobSpawner < ApplicationJob
+  include RedisScanning
+
+  def perform(job_name, *args, key: nil, redis: nil, job_size: nil, **kwargs)
```

Comment:
Let's disable the Metrics/ParameterLists cop (or maybe bump the max to 10).

Reply:
Would you mind if I set `CountKeywordArgs: false` instead? I think there's some value in making sure there aren't too many arguments, but I do think keyword arguments are better because you don't get as confused about "does this value belong in position 4, or position 5?"

Reply:
Not at all, that's actually better.